### ISSUE TITLE
not finding odds info from ESPN, affecting win probability

### ISSUE THREAD

**Issue by ecormany** [saiemgilani/game-on-paper-app#158]
All games are now listing a default disclaimer like this in the win probability section:
> ESPN does not list betting odds for this game, so we've used default values: EMU -2.5, O/U 55.5.

However, the ESPN [gamecast page](https://www.espn.com/college-football/game/_/gameId/401551755) does contain odds.
>Line: USA -17.5
Over/Under: 42.5

Presumably the API data for odds has been altered or removed. Hopefully this can be updated or switched to an alternate source.

The default odds can significantly affect the early part of the win probability graph when teams are not evenly matched.

**Comment by akeaswaran** [comment]
TLDR: This is an artifact of ESPN's move to ESPN BET/Penn Gaming as their main odds provider and we haven't come up with a good solution for it yet. 
--- 

More information: When ESPN moved to ESPN BET, they yanked odds information from the JSON attribute in their game data where our system expects to find it. They now supply that information to their game pages via websockets that we can't access, assumedly so they can deliver live updates. The weird thing is that odds information is still in the scoreboard data endpoint, so really we just need to design a way to access it from there. We just haven't gotten to it yet.
